packages/cli/src/commands/server/launchDebugger.ts

```diff
@@ -46,8 +45,14 @@ function getChromeAppName(): string {
   }
 }
 
-function launchChrome(url: string) {
-  return open(url, {app: [getChromeAppName()], wait: true});
+async function launchChrome(url: string) {
```

Comment:
Please revert this whole change. It introduces a regression, since we don't want to catch this exception here